### PR TITLE
[TASK] Remove zend-json dependency

### PR DESCRIPTION
**Pull request by liayn**
Fixes: #56

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -3,10 +3,42 @@
...
 class Application
 {
+    /**
+     * Sample configuration
+     *
+     * Format for this example is JSON for nice readability
+     *
+     * @var string
+     */
+    private static $configuration = '
```
Comment (compeak) [inline index.php]: Why not use a PHP data structure directly? Is there a point to parsing a json string into a data structure?

Comment (compeak) [inline index.php]: Okay, I overlooked

> for nice readability

But is that really the case?
Wouldn't errors caused by misconfiguration be easier to track down if this was a PHP data structure?

Comment (liayn) [inline index.php]: Because I consider human reading of PHP arrays  for configuration rather tedious.
But if you prefer a straight PHP array, no problem.

**Comment by liayn** [comment]
I have not clue why codeclimate doesn't like my safe access to the GET variable. And the complaint about the else parts is also very funny.

**Comment by liayn** [comment]
The CS fixer results are wrong too. The functions require a `null` return value, hence a simple `return;` is semantically wrong. PHPStorm's inspections color that yellow immediately.

**Comment by liayn** [comment]
How can we get codeclimate and cs-fixer correct?

File: index.php (hunk 1 above)
Comment (liayn) [inline index.php]: > Wouldn't errors caused by misconfiguration be easier to track down if this was a PHP data structure?

That's true as well. I'll convert to PHP array

File: README.md [hunk 2]
```diff
@@ -37,7 +37,9 @@ Installing the Shariff backend on you own server
 
 To run Shariff under a certain URL, unzip the [release](https://github.com/heiseonline/shariff-backend-php/releases) zip file into a directory under the document root of your web server.
 
-This zip file contains a configuration file `shariff.json`. The following configuration options are available:
+Take a look into the demo application in the `index.php` file and adjust the configuration as necessary.
```
Comment (core23) [inline README.md]: Maybe move all demo code to an extra `demo` directory?

**Comment by compeak** [comment]
I "fixed" it by disabling the `empty_return` fixer and ignoring the codeclimate issues.

**Comment by liayn** [comment]
Thanks

**Pull request merged**